project/MiMa.scala

```diff
@@ -46,7 +46,7 @@ object MiMa extends AutoPlugin {
             else Seq.empty
           } ++ akka24StreamVersions ++ akka24WithScala212 ++ akka25Versions
 
-        case "2.12" =>
+        case _ =>
```

Comment:
I do not think the change is correct. Since there is no Akka released for 2.13 `akkaPreviousArtifacts` should return empty list when Scala 2.13 is used, as there are no jars to check BC against.

So instead of this change, add `case "2.13" => Seq.empty`